src/azul/service/async_manifest_service.py

```diff
@@ -61,43 +76,74 @@ class AsyncManifestService(AbstractService):
     """
     Starting and checking the status of manifest generation jobs.
     """
-    step_function_helper = StepFunctionHelper()
+    helper = StepFunctionHelper()
+
+    step_function_execution_retries = 3
 
     def __init__(self, state_machine_name):
         self.state_machine_name = state_machine_name
 
-    def start_generation(self, input: JSON) -> Token:
-        execution_id = str(uuid.uuid4())
-        self.step_function_helper.start_execution(self.state_machine_name,
-                                                  execution_id,
-                                                  execution_input=input)
-        return Token(execution_id=execution_id,
-                     request_index=0,
-                     wait_time=self._get_next_wait_time(0))
+    def start_generation(self, input: JSON, execution_id: str) -> Token:
+
+        for retry_index in range(self.step_function_execution_retries):
+            token = Token(execution_id=f'{execution_id}_{retry_index}',
+                          request_index=0,
+                          wait_time=self._get_next_wait_time(0))
+            try:
+                self.helper.start_execution(self.state_machine_name,
+                                            token.execution_id,
+                                            execution_input=input)
+            except ClientError as e:
+                if e.response['Error']['Code'] == 'ExecutionAlreadyExists':
+                    if retry_index + 1 == self.step_function_execution_retries:
+                        raise StateMachineError('Maximum manifest generation retries reached.',
+                                                token.execution_id)
+                    output, status = self._inspect_generation(token)
+                    if status not in Status:
+                        raise StateMachineError(status, output)
+                    if status is Status.running:
+                        return token.advance(wait_time=self._get_next_wait_time(token.request_index))
+                    else:
+                        # If a manifest expires before the name of the Step
+                        # Function execution that created it can be reused
+                        # (90 days), StartExecution will fail with
+                        # ExecutionAlreadyExists. In that case we need to retry
+                        # even a successful execution, so that it can recreate
+                        # the manifest.
+                        continue
```

Comment:
`continue` skips the remainder of the loop body but there is no remainder so a `pass` should do.